src/lib/y2network/proposal_settings.rb

```diff
@@ -0,0 +1,94 @@
+# encoding: utf-8
+#
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2packager/package"
+
+module Y2Network
+  # Class that stores the proposal settings for network during installation.
+  class ProposalSettings
+    include Yast::Logger
+    include Yast::I18n
+
+    # [Boolean] network service to be used after the installation
+    attr_accessor :backend
+
+    # Constructor
+    def initialize
+      Yast.import "PackagesProposal"
+      Yast.import "Lan"
+
+      @backend = use_network_manager? ? :network_manager : :wicked
+    end
+
+    # Services
```

Comment:
Leftover?

Reply:
Yep, it is not related to services anymore